lib/main.dart

```diff
@@ -7,119 +9,26 @@ void main() {
 class MyApp extends StatelessWidget {
   const MyApp({super.key});
 
+  static const title = 'Movie Card';
+
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       title: 'Flutter Demo',
+      initialRoute: "/",
```

Comment:
Fix initial route, you can refer to the static field of your widget

Reply:
Fixed